### ISSUE TITLE
indeterminate checkbox appears the same as checked in Windows High Contrast Mode - also disabled and invalid are not distinguishable

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#760]
### Expected Behaviour
Should be able to distinguish checkbox states

### Actual Behaviour
Checked and indeterminate appear the same
invalid and disabled cannot be distinguished either

### Reproduce Scenario (including but not limited to)
turn on WHCM
run checkbox demo page
observe states example section

#### Platform and Version
Windows 10
Edge Chromium

#### Logs taken while reproducing problem
![Spectrum Web Components - Profile 1 - Microsoft​ Edge 7_10_2020 5_13_52 PM](https://user-images.githubusercontent.com/1724479/87211922-cdcd6e80-c2d0-11ea-8549-8ec1238cf3a5.png)

**Comment by jnurthen** [comment]
Note indeterminate is ok on Edge Classic but invalid and disabled are still an issue.

**Comment by Westbrook** [comment]
Is this still related to #297 in that this originates upstream? Or have you seen motion of [the issue therein](https://github.com/adobe/spectrum-css/issues/168) that we've yet to adopt?

**Comment by Westbrook** [comment]
@jnurthen Would you be able to confirm whether this is still true with the recent updates to Spectrum CSS here?

**Comment by jnurthen** [comment]
Checkbox should be fixed although I am working on further refinements to better match spectrum as they have now defined high contrast requirements.
Note - all the high contrast work got broken by the t-shirt sizing and will need https://github.com/adobe/spectrum-css/pull/1096 (which is now merged) to make it work again.
